pt-BR/mail/installer/mui.properties

```diff
@@ -54,11 +54,11 @@
 # from en-US contains a \n.
 MUI_TEXT_WELCOME_INFO_TITLE=Bem-vindo ao assistente de instalação do $BrandFullNameDA
 MUI_TEXT_WELCOME_INFO_TEXT=Este assistente conduzirá a instalação do $BrandFullNameDA.\n\nRecomenda-se que você saia de todos os outros aplicativos antes de iniciar a instalação. Assim será possível atualizar arquivos relevantes do sistema sem que seja necessário reiniciar o computador.\n\n$_CLICK
-MUI_TEXT_LICENSE_TITLE=License Agreement
-MUI_TEXT_LICENSE_SUBTITLE=Please review the license terms before installing $(^NameDA).
-MUI_INNERTEXT_LICENSE_TOP=Press Page Down to see the rest of the agreement.
-MUI_INNERTEXT_LICENSE_BOTTOM_CHECKBOX=If you accept the terms of the agreement, click the check box below. You must accept the agreement to install $(^NameDA). $_CLICK
-MUI_INNERTEXT_LICENSE_BOTTOM_RADIOBUTTONS=If you accept the terms of the agreement, select the first option below. You must accept the agreement to install $(^NameDA). $_CLICK
+MUI_TEXT_LICENSE_TITLE=Acordo de Licença
+MUI_TEXT_LICENSE_SUBTITLE=Por favor, leia os termos de licença antes de instalar o $(^NameDA).
+MUI_INNERTEXT_LICENSE_TOP=Pressione a tecla Page Down para ver o restante do acordo.
+MUI_INNERTEXT_LICENSE_BOTTOM_CHECKBOX=Se você aceita os termos do acordo, marque a caixa abaixo. Você deve aceitar o acordo para instalar o $(^NameDA). $_CLICK
+MUI_INNERTEXT_LICENSE_BOTTOM_RADIOBUTTONS=Se você aceita os termos do acordo, seleciona a primeira opção abaixo. Você deve aceitar o acordo para instalar o $(^NameDA). $_CLICK
```

Comment:
CORRECTIONS NEEDED it should be MUI_INNERTEXT_LICENSE_BOTTOM_RADIOBUTTONS=Se você aceita os termos do acordo, selecione a primeira opção abaixo. Você deve aceitar o acordo para instalar o $(^NameDA). $_CLICK